src/data/features.ts

```diff
@@ -97,9 +97,9 @@ export const previewFiles = [
 		name: "Files v3",
 		icon: "/ui/icons/folder.png",
 		items: 16,
-		added: "Wednesday, October 6, 2021 42:69 AM",
-		modified: "Thursday, January 1, 1970 00:00",
```

Comment:
These two were very intentional easter eggs.

Reply:
Okay, I'll return that to it's original state. However, can I spread these Easter eggs out, not have them only on one file; As in, I'll put the 42:69 time on waves.png, and leave the '1970 bug' on Files v3. Also, where is the 'C:\Users\Sara\Documents\index.html' stored? I'd like to add a link to the 'Look at this cool button' to the files discord, if possible.